bundles/scripting/core/pom.xml

```diff
@@ -192,6 +193,11 @@
             <version>2.4</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Gson is not part of Sling yet. I would prefer to just rely on sling.commons.json here.